src/main/lua/exasolrls/adapter_properties.lua

```diff
@@ -73,21 +76,19 @@ end
 --
 -- @raise validation error
 --
--- @return self for fluent programming
---
 function M:validate()
     if not self:has_value(M.SCHEMA_NAME_PROPERTY) then
-        error('F-RLS-PROP-1: Missing mandatory property "' .. M.SCHEMA_NAME_PROPERTY
-            .. '". Please define the name of the source schema.');
+        exaerror.create("F-RLS-PROP-1", "Missing mandatory property '" .. M.SCHEMA_NAME_PROPERTY .. "' ")
+            :add_mitigations("Please define the name of the source schema."):raise()
     end
     if self:is_property_set(M.TABLE_FILTER_PROPERTY) then
         if self:is_empty(M.TABLE_FILTER_PROPERTY) then
-            error("F-RLS-PROP-2: Table filter property must not be empty. Please either remove the property "
-                .. M.TABLE_FILTER_PROPERTY
-                .. " or provide a comma separated list of tables to be included in the Virtual Schema.")
+            exaerror.create("F-RLS-PROP-2", "Table filter property '" .. M.TABLE_FILTER_PROPERTY
+                .. "' must not be empty.")
+                :add_mitigations("Please either remove the property or provide a comma separated list of tables"
+                    .. " to be included in the Virtual Schema."):raise(0)
```

Comment:
Why does this raise have 0 and the previous one is empty?

Reply:
The semantics are `raise(level)` like in the built-in `error(message, level)`. Level zero means: don't display a stack trace.

And now that I think about it, I guess you are right, the other one should suppress the stack trace too!